packages/generator/src/sdk-metadata/generation-and-usage.ts

```diff
@@ -1,43 +1,50 @@
-import { VdmServiceMetadata } from '../vdm-types';
-import { getLinks } from './links';
 import type {
   GenerationAndUsage,
   InstructionWithText
-} from './sdk-metadata-types';
+} from '@sap-cloud-sdk/generator-common';
+import {
+  getSdkVersion,
+  getLinks,
+  apiSpecificUsageText,
+  genericUsageText,
+  getGenerationSteps
+} from '@sap-cloud-sdk/generator-common';
+import { VdmServiceMetadata } from '../vdm-types';
 import { genericGetAllCodeSample } from './code-samples/generic-get-all-code-sample';
-import { getGeneratorVersion } from './pregenerated-lib';
 
 export async function getGenerationAndUsage(
   service: VdmServiceMetadata
 ): Promise<GenerationAndUsage> {
   return {
-    genericUsage: await getGenericUsage(),
-    apiSpecificUsage: await getApiSpecificUsage(service),
+    genericUsage: getGenericUsage(),
+    apiSpecificUsage: getApiSpecificUsage(service),
```

Comment:
I think the links also contain a generation manual which depends if it is OData or OpenApi